doc/ext/sphinx_gallery/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 """
 import os
-__version__ = '0.1.2'
+__version__ = '0.1.7'
```

Comment:
If I understood correctly, the update in this PR contains sphinx-gallery 0.1.7+https://github.com/sphinx-gallery/sphinx-gallery/pull/190 .
If it is indeed so, I'd suggest either to wait for that patch being merged, or to note somewhere the details regarding the version used.